app/components/post_textbox/post_textbox.js

```diff
@@ -27,10 +27,12 @@ import {changeOpacity, makeStyleSheetFromTheme} from 'app/utils/theme';
 const INITIAL_HEIGHT = Platform.OS === 'ios' ? 34 : 36;
 const MAX_CONTENT_HEIGHT = 100;
 const MAX_MESSAGE_LENGTH = 4000;
+const IS_REACTION_REGEX = /(^\+:([^\s]*):)$/i;
```

Comment:
This pattern should include `:` in the character class since emoji names can't contain colons.